src/database/models/requestSessions.js

```diff
@@ -0,0 +1,104 @@
+'use strict'
+module.exports = (sequelize, DataTypes) => {
+	const RequestSession = sequelize.define(
+		'RequestSession',
```

Comment:
@sumanvpacewisdom why we need src/database/models/requestSessionRequest.js file? we have this src/database/models/requestSessions.js